scripts/commands.py

```diff
@@ -91,6 +93,13 @@ def execute_command(command_name, arguments):
             return delete_file(arguments["file"])
         elif command_name == "search_files":
             return search_files(arguments["directory"])
+        ####
+        # Changes 
+        ####
+
+        ####
+        # END CHANGES 
+        ####
```

Comment:
Doesn't look right

Reply:
Yeah this where some comments from my development process.